takeshape/penny
# Generate json-ld on product pages

### Screenshots

![Screen Shot 2022-11-22 at 3 47 08 PM](https://user-images.githubusercontent.com/211047/203443831-954c5b90-535f-4fe8-992f-f2907d07b7a4.png)

### Test Plan (Steps to test):

1. Visit the preview site at https://penny-git-sc-10178-featurepenny-ensure-product-592833-takeshape.vercel.app/
1. Pull up a product page. You can log the json-ld into the console by typing `document.querySelector('#product-json-ld')`
1. Compare the json-ld generated to Google's docs at https://developers.google.com/search/docs/appearance/structured-data/product

### Checklist:
